_config.yml

```diff
@@ -133,52 +133,54 @@ fetch-remote:
     paths:
       - dest: "engine/extend"
         src:
-          - "docs/extend/**"
+          - file: "docs/extend/**"
       - dest: "engine"
         src:
-          - "docs/deprecated.md"
+          - file: "docs/deprecated.md"
       - dest: "engine/reference"
         src:
-          - "docs/reference/run.md"
+          - file: "docs/reference/run.md"
       - dest: "engine/reference/commandline"
         src:
-          - "docs/reference/commandline/cli.md"
-          - "docs/reference/commandline/dockerd.md"
+          - file: "docs/reference/commandline/cli.md"
+          - file: "docs/reference/commandline/dockerd.md"
 
   - repo: "https://github.com/docker/docker"
     default_branch: "master"
     ref: "20.10"
     paths:
       - dest: "engine/api"
         src:
-          - "docs/api/**"
+          - file: "docs/api/**"
 
   - repo: "https://github.com/docker/compose-cli"
     default_branch: "main"
     ref: "main"
     paths:
       - dest: "cloud"
         src:
-          - "docs/*.md"
-          - "!docs/README.md" # readme to make things nice in the compose-cli repo, but meaningless here
-          - "!docs/architecture.md" # Compose-CLI architecture, unrelated to cloud integration
+          - file: "docs/*.md"
+          - file: "!docs/README.md" # readme to make things nice in the compose-cli repo, but meaningless here
+          - file: "!docs/architecture.md" # Compose-CLI architecture, unrelated to cloud integration
 
   - repo: "https://github.com/distribution/distribution"
     default_branch: "main"
     ref: "release/2.7"
     paths:
       - dest: "registry/spec"
         src:
-          - "docs/spec/**"
-          - "!docs/spec/api.md.tmpl"
+          - file: "docs/spec/**"
+          - file: "!docs/spec/api.md.tmpl"
       - dest: "registry"
         src:
-          - "docs/configuration.md"
+          - file: "docs/configuration.md"
 
-  - repo: "https://github.com/moby/buildkit"
+  - repo: "https://github.com/crazy-max/buildkit"
     default_branch: "master"
-    ref: "master"
+    ref: "docs-ref-move-frontmatter"
     paths:
-      - dest: "engine/reference/builder.md"
+      - dest: "_includes/dockerfile/reference.md"
         src:
-          - "frontend/dockerfile/docs/reference.md"
+          - file: "frontend/dockerfile/docs/reference.md"
+            line_start: 2
+            line_end: -1
```

Comment:
@thaJeztah As discussed, removed the `github_content` liquid tag plugin and enhance our `fetch_remote` plugin to be able to specify `line_start` and  `line_end`

Reply:
Argh `edit_url` and `issue_url` is lost then :persevere:

Reply:
Ok managed to fix `edit_url` and `issue_url` inclusion by grabbing the one available in site default scope. I have created a dedicated liquid tag named `include_remote` that does pretty much the same job as `include` except it also handles `<line_start>`, `<line_end>` and creates the right `edit_url` and `issue_url`.

LGTY @thaJeztah ?